Formula/q/qt.rb

```diff
@@ -126,6 +126,26 @@ class Qt < Formula
     sha256 "b36a1c245f2d304965eb4e0a82848379241dc04b865afcc4aab16748587e1923"
   end
 
+  # Backport Chromium changes for libxml 2.12 compatibility
+  # https://chromium.googlesource.com/chromium/src/+/871f8ae9b65ce2679b0bc0be36902d65edf0c1e4
+  patch do
+    url "https://chromium-review.googlesource.com/changes/chromium%2Fsrc~4985186/revisions/4/patch?zip&path=third_party%2Fblink%2Frenderer%2Fcore%2Fxml%2Fxslt_processor.h"
+    directory "qtwebengine/src/3rdparty/chromium"
+    sha256 "61d0352bb43ab796e1702f003f119a0838e5357517ee90f71bcc5cd0ba184e36"
+  end
+  patch do
+    url "https://chromium-review.googlesource.com/changes/chromium%2Fsrc~4985186/revisions/4/patch?zip&path=third_party%2Fblink%2Frenderer%2Fcore%2Fxml%2Fxslt_processor_libxslt.cc"
+    directory "qtwebengine/src/3rdparty/chromium"
+    sha256 "dc8cf92c07cad7da51510d2233968af4b0f408ff6857153ffa1d7387bb77f1b7"
+  end
```

Comment:
Not too sure how stable these Gerrit links are but they do allow grabbing diff of each file in zip format.

Could alternatively use Gentoo/Arch's patch or a corresponding `:DATA` section